packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -738,7 +739,7 @@ It took ${timeInMsToDownloadAllFromCache2}ms to execute download() for all binar
         opts = opts || {}
         // We only make binaries fail with a timeout, not checksums
         if (!String(url).endsWith('.sha256')) {
-          opts.timeout = 1
+          opts.signal = timeoutSignal(1)
```

Comment:
```suggestion
          opts.signal = timeoutSignal(0)
```
Nitpick, to stay consistent